main.go

```diff
@@ -72,3 +72,55 @@ func stripPath(path string) string {
 	dir := filepath.Dir(path)
 	return strings.TrimPrefix(dir, build.Default.GOPATH+"/src/")
 }
+
+// safeRename safely moves files across different file systems.
```

Comment:
I'm not the project's owner, so whatever @FiloSottile prefers.

`os.Rename()` _because_ it renames. In my book, moving implies support for file system, renaming doesn't. There's nothing safe in there, just different semantic.